### PR TITLE
feat: Make options for  FlexibleControls available outside Reveal + Add speed option

### PR DESCRIPTION
**Pull request by nilscognite**
#### Type of change

Feat  

## Description :pencil:

* Expose the options out of Reveal
* Made a new speed option. Default this is 0

## How has this been tested? :mag:

Can not test this yet. Another PR in components

### CODE REVIEW

File: viewer/packages/camera-manager/src/Flexible/FlexibleControlsOptions.ts [hunk 1]
```diff
@@ -117,6 +118,13 @@ export class FlexibleControlsOptions {
     }
   }
 
+  public getKeyboardSpeed(shift: boolean): number {
+    if (!shift) {
```
Comment (danpriori) [inline viewer/packages/camera-manager/src/Flexible/FlexibleControlsOptions.ts:122]: If I understood this correctly, the keyboard speed will be enabled only when the user is pressing shift?
It was requested? I thought that the user only needs to set the speed from the dropdown and just use it. If the user has to keep pressing the shift while wanting to keep the movement faster, is it not be annoying?

**Review by danpriori — APPROVED** [review]
Just a comment.
I think it is good if the "shift" is intentional and we can approve. Otherwise I think we have to change it.

File: viewer/packages/camera-manager/src/Flexible/FlexibleControlsOptions.ts (hunk 1 above)
Comment (nilscognite) [reply viewer/packages/camera-manager/src/Flexible/FlexibleControlsOptions.ts:122]: No. Look at the code.  The control I have made changes this.keyboardSpeed. This used when moving by keyboard. So the user is able to change that. 

If shift is pressed, the speed will be multiplied by  this.keyboardFastMoveFactor.

Comment (danpriori) [reply viewer/packages/camera-manager/src/Flexible/FlexibleControlsOptions.ts:122]: Ah, yeah, right. So we already have a move factor when pressing shift and it is already there from a long time, way before this keyboard speed... you're right.
All good!

**Pull request merged**